editor-components/stats/stats2/stats2.tsx

```diff
@@ -177,38 +130,113 @@ class Stats2Page extends BaseStats {
   }
 
   render() {
+    const cards = this.castToObject<Card[]>("cards");
+
     return (
       <div className={this.decorateCSS("container")}>
         <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("bottom-child")}>
-            {this.castToObject<ICard[]>("card-content").map(
-              (cardData: ICard, indexCard: number) => (
-                <div key={indexCard} className={this.decorateCSS("card")}>
-                  <div className={this.decorateCSS("toppom")}>
-                    <div className={this.decorateCSS("img-container")}>
-                      <img src={cardData.image} alt="" />
-                    </div>
-                    <div className={this.decorateCSS("stats-texts")}>
-                      <h4 className={this.decorateCSS("card-data-title")}>{cardData.title}</h4>
-                      <div className={this.decorateCSS("counts")}>
-                        <p className={this.decorateCSS("count")}>{cardData.description}</p>
-                        <p className={`${cardData.isIncrease ? this.decorateCSS("increase") : this.decorateCSS("decrease")} ${this.decorateCSS("percent")}`}>{cardData.isIncrease ? "↑" : "↓"} {cardData.increaseValue}</p>
-                      </div>
-                    </div>
-                  </div>
-                  <div className={this.decorateCSS("bottoms")}>
-                    <ComposerLink path={cardData.buttonLink}>
-                      <p className={this.decorateCSS("card-button-text")}>{cardData.buttonText}</p>
-                    </ComposerLink>
-                  </div>
-                </div>
-              )
-            )}
+          <div className={this.decorateCSS("header-wrapper")}>
+            <div className={this.decorateCSS("header")}>{this.getPropValue("header")}</div>
+            <div className={this.decorateCSS("subHeader")}>
+              {this.getPropValue("subHeader")}
+              <button className={this.decorateCSS("contact-button")}>{this.getPropValue("contactButton")}</button>
```

Comment:
remove button if `this.getPropValue("contactButton")` value is empty

[check this out](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)